src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1409,6 +1409,16 @@ void NetworkOPsImp::switchLastClosedLedger (
     clearNeedNetworkLedger ();
     newLedger->setClosed ();
     auto openLedger = std::make_shared<Ledger> (false, std::ref (*newLedger));
+    // Apply transactions from the current open ledger to
+    // this new one, including locally held transactions.
+    {
+        // VFALCO Which hash do we use?
```

Comment:
Yes.  

Every single line of this code is new to me - if you're wanting someone who will catch subtle errors you won't, you might consider a review by someone who's familiar with transactions.